src/graph/backend/dnnl/passes/compile_ops.cpp

```diff
@@ -45,12 +49,15 @@ status_t compile_ops(std::shared_ptr<subgraph_t> &sg) {
                 = op_schema_registry_t::get_op_schema(op->get_kind());
         if (!opm) {
             assertm(false, "no schema for current op");
```

Comment:
Do we have you keep the `assertm` after enriching the verbose? If not, you can fuse the `if` condition check into the macro.